.github/workflows/awsfulltest.yml

```
@@ -15,7 +15,7 @@ jobs:
     steps:
       - name: Launch workflow via tower
         uses: nf-core/tower-action@v3
-        # TODO nf-core: You can customise AWS full pipeline tests as required
+        # nf-core: You can customise AWS full pipeline tests as required
```

I think this TODO was added by the nf-core developers. I should not have removed it
```suggestion
        # TODO nf-core: You can customise AWS full pipeline tests as required
```